include/vcpkg/base/system.process.h

```diff
@@ -28,12 +28,8 @@ namespace vcpkg
     struct Command
     {
         Command() = default;
-        explicit Command(const Path& p) { path_arg(p); }
         explicit Command(StringView s) { string_arg(s); }
-        explicit Command(const std::string& s) { string_arg(s); }
-        explicit Command(const char* s) { string_arg({s, ::strlen(s)}); }
 
-        Command& path_arg(const Path& p) & { return string_arg(p); }
         Command& string_arg(StringView s) &;
```

Comment:
```suggestion
        Command& arg(StringView s) &;
```

But I think that would be too invasive for this PR. It would be a nice follow-up though.

Reply:
Will make a separate PR with that controlR controlR :)